spec/ns-obf/obf-procedure.txt

```diff
@@ -71,17 +71,21 @@ Description:       "The justification that the procedure was performed."
 
 Entry:             RadiationProcedure
 Parent:            Procedure
-Description:       "A radiation oncology procedure. The procedure might represent a complete course of treatment, or a single session that is part of a series of treatments. If the treatment is part of a larger course of treatment, the PartOf attribute should indicate that. If the procedure represents the entire course of therapy, the RadiationFractionsDelivered should indicate the number of sessions. If the procedure represents one session, then the RadiationFractionsDelivered would be 1, and the TotalRadiationDoseDelivered will be the same as the RadiationDosePerFraction."
-Property:          RadiationDosePerFraction 0..1
-Property:          RadiationFractionsDelivered 0..1
-Property:          TotalRadiationDoseDelivered 0..1
+Description:       "A radiation oncology procedure. The procedure might represent a complete course of treatment, or a single session that is part of a series of treatments. If the treatment is part of a larger course of treatment, the 'PartOf' attribute should indicate that. If the procedure represents the entire course of therapy, RadiationDose.RadiationFractionsDelivered should indicate the number of sessions. If the RadiationProcedure represents one session, then RadiationDose.RadiationFractionsDelivered should be 1, and the RadiationDose.TotalRadiationDoseDelivered will be the same as the RadiationDose.RadiationDosePerFraction."
```

Comment:
We should probably remove "oncology" since this entry is part of the OBF classes and radiation can be applied to other diseases (e.g.: inflammatory, degernative).  We have CancerRelatedRadiationProcedure which could further narrow the context.